### PR TITLE
Expand context tests

### PR DESCRIPTION
**Pull request by BigBlueHat**
- Fix parentheticals in subsequent items test.
- Move VP create and verify options to outer scope.
- Expand context tests for VPs.

### CODE REVIEW

File: tests/10-vcdm2.js [hunk 1]
```diff
@@ -19,6 +19,13 @@ const baseContextUrl = 'https://www.w3.org/ns/credentials/v2';
 const tag = 'vc2.0';
 const {match} = filterByTag({tags: [tag]});
 
+// used with VP creation and verification throughout
+const createOptions = {challenge};
+const verifyPresentationOptions = {
+  checks: ['proof'],
```
Comment (PatStLouis) [inline tests/10-vcdm2.js:25]:
```suggestion
```

Comment (PatStLouis) [reply tests/10-vcdm2.js:25]: I suggest to remove the checks option.

**Review by PatStLouis — CHANGES_REQUESTED** [review]
Looks great, we are removing the checks options in the verifiers in favor of the implementation's provided options. It would be great to follow this pattern here.

File: tests/10-vcdm2.js (hunk 1 above)
Comment (BigBlueHat) [reply tests/10-vcdm2.js:25]: Is there more needed at this point beyond the removal of that option?

Comment (aljones15) [reply tests/10-vcdm2.js:25]: @PatStLouis @BigBlueHat so DB needs `checks: ['proof']` but other implementers might not need that option and further `options.checks` has not be in the VC API for awhile so the best way to handle this is to add `options.checks` to the DB implementation and do something like this:

https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/6d61d0f3940694edb12437bcb0f457a8b7cb56cb/assertions.js#L25-L46

How to handle this situation in the bitstring status list suite is another issue as those tests will probably need us to expose a second verify with the correct checks for our verifier.

Comment (BigBlueHat) [reply tests/10-vcdm2.js:25]: @PatStLouis definitely looks like we should change this, but I'd prefer to not block this PR for that. I'll make an issue, if you wouldn't mind re-reviewing this one, that'd be great. Thanks!

Comment (aljones15) [reply tests/10-vcdm2.js:25]: @BigBlueHat so the DB verifier now has the checks in the options here: https://github.com/w3c/vc-test-suite-implementations/blob/9a014669b6f62ed9c1a920ef9e054c510abefe6c/implementations/DigitalBazaar.json#L82-L91

so you can now pass the verifier's implementation specific options to each verifier which might result in failure or more success LMK.

Comment (BigBlueHat) [reply tests/10-vcdm2.js:25]: Thanks! I'd still rather do it in a separate PR as doing it here will cause conflicts with all the other pending PRs...

**Review by PatStLouis — APPROVED** [review]
LGTM

File: tests/10-vcdm2.js (hunk 1 above)
Comment (aljones15) [reply tests/10-vcdm2.js:25]: separate PR sounds like the right approach

**Review by aljones15 — APPROVED** [review]
LGTM

**Pull request merged**